Add an example on how to get started with the Python API (#1474)
GaloisInc/cryptol: cryptol-remote-api/python/GettingStarted.md

1. Run the Cryptol remote API server:

    > cryptol-remote-api http /cryptol

> This starts an HTTP server with default settings, which means that

Reviewer: ```suggestion
This starts an HTTP server with the default settings, which means that
```

> the server can be accessed at `http://localhost:8080/cryptol`

Reviewer: A general comment: this sentence should be indented three spaces so that it is at the same level of indentation as the "Run the Cryptol remote API server" sentence above. That way, GitHub will render this text with the same indentation as above. Currently, it looks like this:

![cryptol](https://user-images.githubusercontent.com/2364661/203633017-d5060741-81cf-4e2b-9101-7811a3ec19fd.png)

Similar comments apply to all numbered bullet points below this one.

2. Start a Python shell:

    > poetry run python

> This runs a python shell within an environment setup by

Reviewer: ```suggestion
This runs a Python shell within an environment set up by
```

[poetry](https://python-poetry.org/)

3. Load the `cryptol` library:

  >>> import cryptol

4. Connect to the remote server:

    >>> cry = cryptol.connect(url="http://localhost:8080/cryptol", reset_server=True)

The URL is the one where we started the server in step 1.  The second parameter resets
the state of the server.  See `help("cryptol.connect")` for other options.

> At this point `cry` is an object that we can use to interact with Cryptol.

Reviewer: ```suggestion
At this point, `cry` is an object that we can use to interact with Cryptol.
```

5. Load the standard library:

    >>> cry.load_module("Cryptol")

The `load_module` method is similar to the `:module` command of the Cryptol REPL---it
loads a module to the server's state.  The module `Cryptol` is the standard library for
Cryptol.

6. Evaluate an expression:

    >>> it = cry.evaluate_expression("1+1")

> This command sends the expression `1 + 1` to the server and the value `it` contains

Reviewer: ```suggestion
This command sends the expression `1 + 1` to the server, and the value `it` contains
```
